### ISSUE TITLE
[Feature Request / Suggestion]: page speed to be calculated in words per minute

### ISSUE THREAD

**Issue by puntofisso** [manifestinteractive/teleprompter#66]
### Suggestion / Feature Request

Hi, I know this might be tricky to implement, but the page speed could be expressed in words per minutes, instead of 0-50. This is particularly useful if someone is writing a speech, knowing how many words they need to write.

**Comment by manifestinteractive** [comment]
I was looking into this when I first built the app and ran into quite a few roadblocks that made this impossible.

The primary issue is that even the exact same font is rendered differently by different operating systems. Heck, eve different browsers on the same OS have slight differences in kerning and line height.

This makes getting something close to accurate for WPM just horribly inaccurate.

Throw into the mix the ability to change font size, and the difference with how browsers handle scroll speed as well, the only real choice you are left with is some generic speed adjuster like I've got now.

What I've heard works for others needing WPM measurements is just to measure your results at different specs to get a feel for what WPM is for you personally.

I say personally too, as people have different talking speeds, or even different cadences they take on during reading which can drastically change WPM even when the same person reads.  Voice actors especially are dynamic to where WPM would 100% depend on how they wanted to do that read. 

So speed setting it is, as it turned out to be the simplest way to handle all that complexity.
